frameworks/cassandra/universe/config.json

```diff
@@ -51,11 +51,21 @@
           "enum": ["serial", "parallel"],
           "default": "serial"
         },
-        "virtual_networks": {
+        "virtual_network_enabled": {
           "description": "Enable DC/OS Virtual Networking",
           "type": "boolean",
           "default": false
         },
+        "virtual_network_name": {
+          "description": "Name of virtual network to join",
+          "type": "string",
+          "default": "dcos"
+        },
+        "virtual_network_plugin_labels": {
+          "description": "Labels to pass to the virtual network plugin. Comma-separated key:value pairs. For example [k_0:v_0,k_1:v_1,...,k_n:v_n]",
```

Comment:
OK, another question @ArtRand. I see from `AttributeStringUtilsTest` that for other key-value like parameters we sometimes use `;` as a separator instead of `,`. Should I update the networking spec to match that too?